interviewing/readme.md

```diff
@@ -0,0 +1,66 @@
+# Interviewing 🧑‍💻
+
+Acing job interviews is a skill that can be learned like any other. Today we will pair up to practice interviewing.
+
+## Objective 💬
+
+At the end of this workshop, participants can:
+
+- [] Introduce themselves in 2 minutes
+- [] Answer soft skill interview questions using the STAR method
+- [] Prepare a strategy to get ready for  technical interviews
+
+## Setup 🌼
+
+- Pair up into groups of 2-3
+
+## Stage 1: Introduce Yourself 🙆
+
+In almost every interview, the interviewer will ask you to introduce yourself. The goal is to give the interviewer a snapshot of who you are, your professional background, and why you're a good fit for the role. Take 5 mins to craft a compelling narrative. 
+
+Some tips:
+- Your response should be 1-2 minutes
+- Try to structure your response, as an example the Present-Past-Future model:
```

Comment:
This could be a good energiser too. Could you contribute it here? https://github.com/CodeYourFuture/curriculum/tree/main/common-content/en/energisers

Reply:
Added [here](https://github.com/CodeYourFuture/curriculum/pull/917). I changed the energiser to be a 30 second elevator pitch due to time constraints.